GiEnJul/ClientApp/src/components/registerGiver/IGiverInputs.tsx

```diff
@@ -0,0 +1,9 @@
+interface IGiverInputs {
```

Comment:
In my opinion the name of this object does not clearly represent its use. 
Maybe it could be renamed to contain Post/Form/Data.. or something like that.
IGiverFormData might be an option.

Also, should these values be nullable?

Reply:
Agree! Also these should not be nullable 👍